work-scheduler-service/src/main/java/com/github/gaalgergo/workscheduler/service/domain/VacationRequest.java

```diff
@@ -0,0 +1,15 @@
+package com.github.gaalgergo.workscheduler.service.domain;
+
+import com.github.gaalgergo.workscheduler.data.entities.EmployeeEntity;
+import lombok.Data;
+
+import java.time.LocalDate;
+
+@Data
+public class VacationRequest {
+
+    private EmployeeEntity employeeEntity;
```

Comment:
This Java bean should aggregate an `Employee` instance instead of the lower-level `EmployeeEntity` object.